common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -669,19 +669,29 @@ def get_transcripts_info(self, is_bumper=False):
         """
         Returns a transcript dictionary for the video.
         """
+        transcripts = self.transcripts
+        sub = self.sub
         if is_bumper:
             transcripts = copy.deepcopy(get_bumper_settings(self).get('transcripts', {}))
             sub = transcripts.pop("en", "")
-        else:
```

Comment:
Which I understand from this is, problem is caused by initialization in else part where `self.transcripts` is not found so `transcripts` is set to `None`. When `items()` is called on `None` object, exception is raised. My question here is why you didn't control this initialization? like `transcripts = self.transcripts if self.transcripts else {}` would have solve this problem, isn't it?